src/qt/bitcoin.cpp

```diff
@@ -259,7 +259,7 @@ void BitcoinApplication::createPaymentServer()
 
 void BitcoinApplication::createOptionsModel(bool resetSettings)
 {
-    optionsModel = new OptionsModel(this, resetSettings);
+    optionsModel = new OptionsModel(node(), this, resetSettings);
```

Comment:
Ok, leave it as is for the purposes of this PR.

There was some subtle bug recently caused by some not-obvious dependency during startup, but the difference with this code here is that here we have an `assert()`, so it will not remain unnoticed.